jenkins/opensearch/benchmark-test.jenkinsfile

```diff
@@ -0,0 +1,305 @@
+                            ingestCount: INGEST_NODE_COUNT,
+                            mlCount: ML_NODE_COUNT,
+                            userTag: "${USER_TAGS},security-enabled:true",
+                            workloadParams: WORKLOAD_PARAMS,
+                            additionalConfig: ADDITIONAL_CONFIG,
+                            dataStorage: DATA_NODE_STORAGE,
+                            mlStorage: ML_NODE_STORAGE,
+                            jvmSysProps: JVM_SYS_PROPS,
+                            captureNodeStat: CAPTURE_NODE_STAT
+                            )
+                            lib.jenkins.Messages.new(this).add(JOB_NAME,
+                                lib.jenkins.Messages.new(this).get([JOB_NAME]) +
+                                    "\nBenchmark tests with security for ${BUILD_ID} completed")
+                        }
+                    }
+                    post {
+                        success {
+                            postCleanup()
+                        }
+                        failure {
+                            postCleanup()
+                        }
+                    }
+                }
+                stage('test-without-security') {
+                agent { label AGENT_LABEL }
+                    steps {
+                        script {
+                            def bundleManifestObj = downloadBuildManifest(
+                                url: BUNDLE_MANIFEST_URL,
+                                path: BUNDLE_MANIFEST
+                            )
+                            echo "BUNDLE_MANIFEST: ${BUNDLE_MANIFEST}"
+                            echo "BUILD_ID: ${BUILD_ID}"
+                            echo "Architecture: ${ARCHITECTURE}"
+
+                            runBenchmarkTestScript(
+                            bundleManifest: BUNDLE_MANIFEST,
+                            workload: TEST_WORKLOAD,
+                            insecure: "true",
+                            singleNode: SINGLE_NODE_CLUSTER,
+                            minDistribution: MIN_DISTRIBUTION,
+                            useFiftyPercentHeap: USE_50_PERCENT_HEAP,
+                            suffix: "${BUILD_NUMBER}",
+                            managerCount: MANAGER_NODE_COUNT,
+                            dataCount: DATA_NODE_COUNT,
+                            clientCount: CLIENT_NODE_COUNT,
+                            ingestCount: INGEST_NODE_COUNT,
+                            mlCount: ML_NODE_COUNT,
+                            userTag: "${USER_TAGS},security-enabled:false",
+                            workloadParams: WORKLOAD_PARAMS,
+                            additionalConfig: ADDITIONAL_CONFIG,
+                            dataStorage: DATA_NODE_STORAGE,
+                            mlStorage: ML_NODE_STORAGE,
+                            jvmSysProps: JVM_SYS_PROPS,
+                            captureNodeStat: CAPTURE_NODE_STAT
```

Comment:
In case you mean `userTag: "${USER_TAGS},security-enabled:false",` then it has to be like this as it is passed as is to `test.sh` and then to python code as is, spacing causes error while reading from command line in python code.

Reply:
I mean to say spacing for these:
```
runBenchmarkTestScript(
bundleManifest: BUNDLE_MANIFEST,
workload: TEST_WORKLOAD,
```

to 

```

runBenchmarkTestScript(
    bundleManifest: BUNDLE_MANIFEST,
    workload: TEST_WORKLOAD,

```

Reply:
got it, updated.

Reply:
Thanks.